components/file_attachment_list/file_attachment_list.test.tsx

```diff
@@ -32,7 +37,21 @@ describe('FileAttachmentList', () => {
             ...baseProps,
             fileCount: 1,
             fileInfos: [
-                {id: 'file_id_1', name: 'file.txt', extension: 'txt'},
+                {
```

Comment:
Could use `getFileInfoMock` here too if you wanted

Reply:
Oops - done!